pyproject.toml

```diff
@@ -0,0 +1,156 @@
+# https://peps.python.org/pep-0517/
+[build-system]
+requires = ["setuptools>=45", "wheel"]
+build-backend = "setuptools.build_meta"
+
+# https://peps.python.org/pep-0621/
```

Comment:
this link is particularly important, this is the PEP that defined how project metadata goes in pyproject.toml